插件在 Koishi 4.17.1 无法启动

报错为 `config TypeError: koishi_1.Context.service is not a function`

复现流程
yarn create koishi
前往插件市场安装此插件
前往插件配置添加此插件
观察日志

<details><summary>完整复现日志</summary>

```
PS C:\Users\5800h\Desktop\temp> yarn create koishi
yarn create v1.22.21
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...

success Installed "create-koishi@6.3.1" with binaries:
      - create-koishi
[##########################] 26/26
  Create Koishi  v6.3.1

√ Project name: ... k3
  Scaffolding project in k3 ...
  Done.

√ Install and start it now? ... yes
➤ YN0000: · Yarn 4.1.0
➤ YN0000: ┌ Resolution step
➤ YN0085: │ + @koishijs/client@npm:5.28.0, @koishijs/plugin-actions@npm:0.0.2, and 669 more.
➤ YN0000: └ Completed in 2s 82ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ 3 packages were added to the project (+ 3.19 MiB).
➤ YN0000: └ Completed in 0s 667ms
➤ YN0000: ┌ Link step
➤ YN0007: │ esbuild@npm:0.18.20 must be built because it never has been before or the last one failed
➤ YN0007: │ esbuild@npm:0.19.12 must be built because it never has been before or the last one failed
➤ YN0007: │ vue-demi@npm:0.14.7 [43011] must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 6s 374ms
➤ YN0000: · Done in 9s 228ms
2024-03-04 05:40:32 [I] app Koishi/4.17.1
2024-03-04 05:40:32 [I] loader apply plugin group:entry
2024-03-04 05:40:32 [I] loader apply plugin group:server
2024-03-04 05:40:32 [I] loader apply plugin server:5y8qa6
2024-03-04 05:40:32 [I] loader apply plugin group:basic
2024-03-04 05:40:32 [I] loader apply plugin commands:fiet0j
2024-03-04 05:40:32 [I] loader apply plugin help:ui55n9
2024-03-04 05:40:32 [I] loader apply plugin locales:be10sy
2024-03-04 05:40:32 [I] loader apply plugin proxy-agent:14o4yr
2024-03-04 05:40:32 [I] loader apply plugin rate-limit:llwemo
2024-03-04 05:40:32 [I] loader apply plugin telemetry:trcxhk
2024-03-04 05:40:32 [I] loader apply plugin group:console
2024-03-04 05:40:32 [I] loader apply plugin actions:zdoyqd
2024-03-04 05:40:32 [I] loader apply plugin analytics:n5ifoe
2024-03-04 05:40:32 [I] loader apply plugin config:ycl320
2024-03-04 05:40:32 [I] loader apply plugin console:lonkfa
2024-03-04 05:40:32 [I] loader apply plugin dataview:ljd2ka
2024-03-04 05:40:32 [I] loader apply plugin explorer:tzddvt
2024-03-04 05:40:32 [I] loader apply plugin logger:6kpqmt
2024-03-04 05:40:32 [I] loader apply plugin insight:anpblv
2024-03-04 05:40:32 [I] loader apply plugin market:g7o2dr
2024-03-04 05:40:33 [I] loader apply plugin notifier:vnjwab
2024-03-04 05:40:33 [I] loader apply plugin oobe:6rpik9
2024-03-04 05:40:33 [I] loader apply plugin sandbox:xlv3cb
2024-03-04 05:40:33 [I] loader apply plugin status:mwahl4
2024-03-04 05:40:33 [I] loader apply plugin theme-vanilla:ors6qo
2024-03-04 05:40:33 [I] loader apply plugin group:storage
2024-03-04 05:40:33 [I] loader apply plugin database-sqlite:fvahl5
2024-03-04 05:40:33 [I] loader apply plugin assets-local:vn3uwi
2024-03-04 05:40:33 [I] loader apply plugin group:adapter
2024-03-04 05:40:33 [I] assets missing config "selfUrl", fallback to "file:" scheme
2024-03-04 05:40:33 [I] server server listening at http://127.0.0.1:5141
2024-03-04 05:40:33 [I] console webui is available at http://127.0.0.1:5141
2024-03-04 05:40:33 [I] sqlite auto creating table user
2024-03-04 05:40:33 [I] sqlite auto creating table binding
2024-03-04 05:40:33 [I] sqlite auto creating table channel
2024-03-04 05:40:33 [I] sqlite auto creating table analytics.message
2024-03-04 05:40:33 [I] sqlite auto creating table analytics.command
2024-03-04 05:40:42 [I] market Yarn 4.1.0
2024-03-04 05:40:42 [I] market ┌ Resolution step
2024-03-04 05:40:43 [I] market + koishi-plugin-jieba@npm:0.3.4, agent-base@npm:6.0.2, follow-redirects@npm:1.15.5, https-proxy-agent@npm:5.0.1, nodejs-file-downloader@npm:4.12.1, and 3 more.
2024-03-04 05:40:43 [I] market └ Completed in 0s 846ms
2024-03-04 05:40:43 [I] market ┌ Fetch step
2024-03-04 05:40:44 [I] market └ Completed in 0s 522ms
2024-03-04 05:40:44 [I] market ┌ Link step
2024-03-04 05:40:44 [I] market └ Completed in 0s 337ms
2024-03-04 05:40:44 [I] market Done in 1s 826ms
2024-03-04 05:40:45 [E] market Error: Cannot find module 'koishi-plugin-jieba'
                        Require stack:
                        - C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\plugin-market\lib\node\index.js
                        - C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\index.js
                        - C:\Users\5800h\Desktop\temp\k3\node_modules\koishi\lib\worker\index.js
                            at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
                            at Function.resolve (node:internal/modules/helpers:187:19)
                            at Proxy.install (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\plugin-market\lib\node\index.js:329:23)
                            at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
                            at async ctx2.console.addListener.authority (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\plugin-market\lib\node\index.js:562:20)
                            at async WebSocket.receive (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\console\lib\index.js:127:21)
2024-03-04 05:40:45 [I] app trigger full reload
2024-03-04 05:40:45 [I] app Koishi/4.17.1
2024-03-04 05:40:45 [I] loader apply plugin group:entry
2024-03-04 05:40:45 [I] loader apply plugin group:server
2024-03-04 05:40:45 [I] loader apply plugin server:5y8qa6
2024-03-04 05:40:45 [I] loader apply plugin group:basic
2024-03-04 05:40:45 [I] loader apply plugin commands:fiet0j
2024-03-04 05:40:45 [I] loader apply plugin help:ui55n9
2024-03-04 05:40:45 [I] loader apply plugin locales:be10sy
2024-03-04 05:40:45 [I] loader apply plugin proxy-agent:14o4yr
2024-03-04 05:40:45 [I] loader apply plugin rate-limit:llwemo
2024-03-04 05:40:45 [I] loader apply plugin telemetry:trcxhk
2024-03-04 05:40:45 [I] loader apply plugin group:console
2024-03-04 05:40:45 [I] loader apply plugin actions:zdoyqd
2024-03-04 05:40:45 [I] loader apply plugin analytics:n5ifoe
2024-03-04 05:40:45 [I] loader apply plugin config:ycl320
2024-03-04 05:40:45 [I] loader apply plugin console:lonkfa
2024-03-04 05:40:45 [I] loader apply plugin dataview:ljd2ka
2024-03-04 05:40:45 [I] loader apply plugin explorer:tzddvt
2024-03-04 05:40:45 [I] loader apply plugin logger:6kpqmt
2024-03-04 05:40:45 [I] loader apply plugin insight:anpblv
2024-03-04 05:40:45 [I] loader apply plugin market:g7o2dr
2024-03-04 05:40:45 [I] loader apply plugin notifier:vnjwab
2024-03-04 05:40:45 [I] loader apply plugin oobe:6rpik9
2024-03-04 05:40:45 [I] loader apply plugin sandbox:xlv3cb
2024-03-04 05:40:45 [I] loader apply plugin status:mwahl4
2024-03-04 05:40:45 [I] loader apply plugin theme-vanilla:ors6qo
2024-03-04 05:40:45 [I] loader apply plugin group:storage
2024-03-04 05:40:45 [I] loader apply plugin database-sqlite:fvahl5
2024-03-04 05:40:45 [I] loader apply plugin assets-local:vn3uwi
2024-03-04 05:40:45 [I] loader apply plugin group:adapter
2024-03-04 05:40:45 [I] assets missing config "selfUrl", fallback to "file:" scheme
2024-03-04 05:40:45 [I] server server listening at http://127.0.0.1:5141
2024-03-04 05:40:45 [I] console webui is available at http://127.0.0.1:5141
2024-03-04 05:40:52 [W] config failed to load jieba
2024-03-04 05:40:52 [W] config TypeError: koishi_1.Context.service is not a function
                            at Object.<anonymous> (C:\Users\5800h\Desktop\temp\k3\node_modules\koishi-plugin-jieba\lib\index.js:163:18)
                            at Module._compile (node:internal/modules/cjs/loader:1376:14)
                            at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
                            at Module.load (node:internal/modules/cjs/loader:1207:32)
                            at Module._load (node:internal/modules/cjs/loader:1023:12)
                            at Module.require (node:internal/modules/cjs/loader:1235:19)
                            at require (node:internal/modules/helpers:176:18)
                            at Proxy.import (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\index.js:143:12)
                            at Proxy.resolve (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\shared.js:262:51)
                            at Proxy.parseExports (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\plugin-config\lib\shared\index.js:86:45)
2024-03-04 05:40:53 [I] loader apply plugin jieba:b50ku8
2024-03-04 05:40:53 [E] loader TypeError: koishi_1.Context.service is not a function
                            at Object.<anonymous> (C:\Users\5800h\Desktop\temp\k3\node_modules\koishi-plugin-jieba\lib\index.js:163:18)
                            at Module._compile (node:internal/modules/cjs/loader:1376:14)
                            at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
                            at Module.load (node:internal/modules/cjs/loader:1207:32)
                            at Module._load (node:internal/modules/cjs/loader:1023:12)
                            at Module.require (node:internal/modules/cjs/loader:1235:19)
                            at require (node:internal/modules/helpers:176:18)
                            at Proxy.import (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\index.js:143:12)
                            at Proxy.resolve (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\shared.js:262:51)
                            at Proxy.forkPlugin (C:\Users\5800h\Desktop\temp\k3\node_modules\@koishijs\loader\lib\shared.js:282:31)
```
</details>

System:
OS: Windows 10 10.0.19045
CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
Binaries:
Node: 20.11.0
Yarn: 4.1.0
Koishi:
Core: 4.17.1
Console: 5.28.0

--- a/src/index.ts
+++ b/src/index.ts
@@ -197,7 +197,6 @@ export namespace Jieba {
   });
 }
 
-Context.service('jieba', Jieba);
 export default Jieba;
 class UnsupportedError extends Error {
   constructor(message: string) {

